oai-mods-add-thumbnail-url.xsl

```diff
@@ -41,13 +41,13 @@
                 <xsl:attribute name="usage">
                     <xsl:value-of select="'primary display'"/>
                 </xsl:attribute>
-                <xsl:value-of select="following::mods:identifier[starts-with(.,'http://')]"/>
+                <xsl:value-of select="following::mods:identifier[starts-with(.,'https://')]"/>
```

Comment:
Is it me, or does it make more sense to do this:

```
<xsl:value-of select="following::mods:identifier[starts-with(.,'http')]"/>
```

This seems like a safer bet, but maybe it doesn't matter.  Thoughts?  If you disagree, I'll approve, just want to ask first.

Reply:
I definitely don't see a problem with this.